subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/JvmModelingServices.java

```diff
@@ -34,17 +34,6 @@ public interface JvmModelingServices {
      */
     Configuration createOutgoingElements(String name, Action<? super OutgoingElementsBuilder> configuration);
 
-    /**
-     * Creates an incoming configuration for resolution of a dependency graph.
-     *
-     * The action is used to configure the created <i>resolvable</i> configuration.
-     *
-     * @param name the name of the configuration
-     * @param action the configuration of the resolvable configuration
-     * @return the resolvable configuration
-     */
-    Configuration createResolvableConfiguration(String name, Action<? super ResolvableConfigurationBuilder> action);
```

Comment:
I'm worried about deleting this. Have we run smoke tests? We often have plugins using these internal builders / helper methods

Reply:
I've run so, so, so many smoke tests on this PR. 😄  If it hasn't broken by now, it's an unnecessary internal method able to be removed.